graphitron-codegen-parent/graphitron-java-codegen/src/main/java/no/sikt/graphitron/generators/db/FetchMappedObjectDBMethodGenerator.java

```diff
@@ -45,24 +47,46 @@ public MethodSpec generate(ObjectField target) {
         // Note that this must happen before alias declaration.
         var selectRowBlock = getSelectRowOrField(target, context);
         var whereBlock = formatWhereContents(context, resolverKeyParamName, isRoot, target.isResolver());
-
+        var fieldHasTableService = target.hasTableServiceDirective();
+        if (fieldHasTableService) {
+            createServiceDependency(target);
+        }
         var querySource = context.renderQuerySource(getLocalTable());
-
         var refContext = target.isResolver() ? context.nextContext(target) : context;
         var actualRefTable = refContext.getTargetAlias();
         var actualRefTableName = refContext.getTargetTableName();
-
         var selectAliasesBlock = createAliasDeclarations(context.getAliasSet());
-
         var orderFields = !LookupHelpers.lookupExists(target, processedSchema) && (target.isIterableWrapped() || target.hasForwardPagination() || !isRoot)
                 ? createOrderFieldsDeclarationBlock(target, actualRefTable, actualRefTableName)
                 : CodeBlock.empty();
 
+        var reservedNames = Stream.of(GraphQLReservedName.PAGINATION_BEFORE,
+                GraphQLReservedName.PAGINATION_AFTER,
+                GraphQLReservedName.PAGINATION_LAST,
+                GraphQLReservedName.PAGINATION_FIRST).map(GraphQLReservedName::getName).toList();
+
+        var tableServiceArgs = fieldHasTableService ? target.getArguments()
+                .stream()
+                .map(AbstractField::getName).
+                filter(name -> !reservedNames.contains(name))
+                .collect(Collectors.collectingAndThen(Collectors.joining(", "),
+                        args -> args.isEmpty() ? CodeBlock.empty() : CodeBlock.of(", $L", args)))
```

Comment:
På dette punktet er det like gjerne å bare kjøre join
```suggestion
                .collect(Collectors.joining(", "))
```